src/js/modules/infragistics.ui.editors.js

```diff
@@ -4478,6 +4486,11 @@
 				this.options.minDecimals;
 		},
 		_setInitialValue: function (value) { // NumericEditor
+			if (typeof value === "undefined") {
```

Comment:
Indeed I was wondering if this should default to nothing or 0 - the unmodified behavior with wrong `nullValue` on the numeric would be empty string, no errors on typing, but unable to clear value which defaults to 0 instead. Will update the initial to stay empty as test definitions match that, but have a leftover `value: ""`.. 😞 
I was wondering if this should default to nothing or 0